Lib/test/test_tarfile.py

```diff
@@ -722,6 +722,28 @@ def format_mtime(mtime):
             tar.close()
             os_helper.rmtree(DIR)
 
+    @staticmethod
+    @unittest.mock.patch("tarfile.data_filter", wraps=tarfile.data_filter)
+    @unittest.mock.patch("tarfile.tar_filter", wraps=tarfile.tar_filter)
+    @unittest.mock.patch("tarfile.fully_trusted_filter", wraps=tarfile.fully_trusted_filter)
+    def test_extractall_default_filter(mock_ft_filter: unittest.mock.Mock,
+                                       mock_tar_filter: unittest.mock.Mock,
+                                       mock_data_filter: unittest.mock.Mock):
+        tar = tarfile.open(tarname, encoding="iso8859-1")
+        DIR = os.path.join(TEMPDIR, "extractall_default_filter")
+        os.mkdir(DIR)
+        try:
```

Comment:
Why not use a context manager as in the previous tests for deprecation?

Reply:
Ah, didn't notice that. I copied the test above this one (`test_extractall`) which uses the try/finally pattern. I'll update both tests to use the context manager.

Reply:
Just modify your test, do not touch others unless it's not related. That way the diff is smaller. 

In the future, we might want to just refactor all tests but that's bordering a cosmetic PR.

Reply:
Makes sense. Reverted.